jupyterhub/values.yaml

```diff
@@ -387,6 +387,7 @@ singleuser:
     pullPolicy:
     pullSecrets: []
   startTimeout: 300
+  k8sApiRequestTimeout: 3
```

Comment:
```suggestion
  k8sApiRequestTimeout:
```

Reply:
I want kubespawners default value to be respected as long as we don't see a reason for this Helm chart to override it. I think that improves maintainability.